examples/step-67/doc/intro.dox

```diff
@@ -2,13 +2,695 @@
 <br>
 
 <i>
-This program was contributed by Martin Kronbichler.
+This program was contributed by Martin Kronbichler. Many ideas presented here
+are the result of common code development with Niklas Fehn, Katharina Kormann,
+Peter Munch and Svenja Schoeder.
+
+This work was partly supported by the German Research Foundation (DFG) through
+the project "High-order discontinuous Galerkin for the exa-scale" (ExaDG)
+within the priority program "Software for Exascale Computing" (SPPEXA).
 </i>
 
 <a name="Intro"></a>
 <h1>Introduction</h1>
 
-This tutorial program implements an explicit time integrator with the
-matrix-free framework using high-order discontinuous Galerkin methods
-for the Euler equations.
+This tutorial program solves the Euler equations of fluid dynamics using an
+explicit time integrator with the matrix-free framework applied to a
+high-order discontinuous Galerkin discretization in space. For details about
+the Euler system and an alternative implicit approach, we also refer to the
+step-33 tutorial program.
+
+<h3>The Euler equations</h3>
+
+The Euler equations are a conservation law, describing the motion of a
+compressible inviscid gas,
+@f[
+\frac{\partial \mathbf{w}}{\partial t} + \nabla \cdot \mathbf{F}(\mathbf{w}) =
+\mathbf{G}(\mathbf w),
+@f]
+where the $d+2$ components of the solution vector are $\mathbf{w}=(\rho, \rho
+u_1,\ldots,\rho u_d,E)^{\mathrm T}$. Here, $\rho$ denotes the fluid density,
+${\mathbf u}=(u_1,\ldots u_d)^\mathrm T$ the fluid velocity, and $E$ the
+energy density of the gas. The velocity is not directly solved for, but rather
+the variable $\rho \mathbf{u}$, the linear momentum (since this is the
+conserved quantity.
+
+The Euler flux function, a $(d+2)\times d$ matrix, is defined as
+@f[
+  \mathbf F(\mathbf w)
+  =
+  \begin{pmatrix}
+  \rho \mathbf{u}\\
+  \rho \mathbf{u} \otimes \mathbf{u}\\
```

Comment:
I guess the pressure term is missing here for the diagonal terms in the velocity.

Reply:
Yes indeed - I fixed it now.